.github/workflows/cover.yaml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
```

Comment:
@alexeldeib want to revert this and we can merge the PR just for the master branch for now?

Reply:
yep, updated